models/snowplow/model.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW {schema}.events as (
```

Comment:
i'm extremely pro lower-case for all reserved and non-reserved words. but agree, we should have a standard and adhere to it.